libs/ledger-live-common/src/bridge/generic-alpaca/prepareTransaction.ts

```diff
@@ -57,7 +57,7 @@ export function genericPrepareTransaction(
         assetOwner,
         customFees: {
           parameters: {
-            fees: new BigNumber(fees.toString()),
+            fees: customParametersFees ? new BigNumber(fees.toString()) : undefined,
           },
         },
       };
```

Comment:
we are doing `await estimateFees` twice line 40 and line 66 !
can't we just do it once ?